homeassistant/components/syncthru/strings.json

```diff
@@ -12,7 +12,7 @@
     "step": {
       "confirm": {
         "data": {
-          "name": "[%key:component::syncthru::config::step::user::data::name%]",
+          "name": "[%key:common::config_flow::data::name%]",
           "url": "[%key:component::syncthru::config::step::user::data::url%]"
```

Comment:
I think this can be updated as well

Reply:
It is not necessary. This is a reference to a string and is therefore displayed correctly.
https://app.lokalise.com/project/130246255a974bd3b5e8a1.51616605/?k=50346678